tests/integration/test_workflow_lifecycle.py

```diff
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+"""Integration tests for workflow lifecycle (start, data updates, stop)."""
+
+import pytest
+
+from ess.livedata.config.workflow_spec import WorkflowId
+from ess.livedata.handlers.monitor_workflow_specs import MonitorDataParams
+
+from .conftest import IntegrationEnv
+from .helpers import wait_for_workflow_job_data, wait_for_workflow_job_statuses
+
+
+@pytest.mark.integration
+@pytest.mark.services('monitor')
+def test_workflow_can_start_and_receive_data(integration_env: IntegrationEnv) -> None:
+    """
+    Test that a workflow can be started and receives data from backend services.
+
+    This test demonstrates the basic integration test pattern:
+    1. Start a workflow via WorkflowController
+    2. Process messages by calling backend.update()
+    3. Wait for workflow response using helpers
+    4. Wait for data to arrive
+    """
+    backend = integration_env.backend
+
+    # Define workflow parameters for monitor histogram workflow
+    workflow_id = WorkflowId(
+        instrument='dummy',
+        namespace='monitor_data',
+        name='monitor_histogram',
+        version=1,
+    )
```

Comment:
(Based on guidelines on test separation. This is not actually relevant with function-scope fixtures.) Both tests use the same id. As far as I can tell, workflows are not removed at the end of a test. Also, do you forbid using pytest-xdist?